### PR TITLE
Update core.py for changes to simple_route

### PR DESCRIPTION
**Pull request by janddd**
The simple_function missed a closing semicolon when intermediate coordinates are used, also the input geometry-format was not consistently set to lowercase in the if statements.

(This is my first pull request, so I hope I'm doing this right.)

### CODE REVIEW

**Comment by ustroetz** [comment]
@mthh or @zifeo could you please have a look at this?

File: osrm/core.py [hunk 1]
```diff
@@ -221,9 +221,9 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
         elif geometry in ("polyline", "geojson") and output == "routes":
             return parsed_json["routes"]
         else:
-            if geometry == "wkb":
+            if geometry.lower() == "wkb":
                 func = Geometry.ExportToWkb
-            elif geometry == "wkt":
+            elif geometry.lower() == "wkt":
```
Comment (zifeo) [inline osrm/core.py]: Could you also correct it [here](https://github.com/ustroetz/python-osrm/blob/master/osrm/core.py#L497)?

File: osrm/core.py [hunk 2]
```diff
@@ -186,7 +186,7 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
         if coord_intermediate:
             url.append(";".join(
                 [','.join([str(i), str(j)]) for i, j in coord_intermediate]))
-
+            url.extend(";")
```
Comment (zifeo) [inline osrm/core.py]: `extend` works here (single character), but it should be `append`

**Comment by zifeo** [comment]
@ustroetz The build is failing due to GDAL, can you check?

**Comment by mthh** [comment]
@zifeo sorry for the late response, I'll try to fix the travis configuration soon !

**Comment by ustroetz** [comment]
I triggered the build again. If it is green, we can merge this too.

**Comment by mthh** [comment]
@ustroetz I guess this needs to be rebased again master now for this PR to take #31 into account.

**Comment by mthh** [comment]
@ustroetz I rebased @janddd patch-1 branch on your master branch (and travis isn't complaining anymore) so i guess we can merge it too ?

**Pull request merged**